documentation_template_latex/zubaxdoc.cls

```diff
@@ -140,16 +141,16 @@
     \centering\huge\bfseries
 }
 
-\titleformat{\chapter}{\huge\bfseries}{}{1em}{\makebox[17mm][l]{\thechapter}\hangindent=17mm}
-\titleformat{\section}{\Large\bfseries}{}{1em}{\makebox[17mm][l]{\thesection}\hangindent=17mm}
-\titleformat{\subsection}{\bfseries}{}{1em}{\makebox[17mm][l]{\thesubsection}\hangindent=17mm}
+\titleformat{\chapter}{\huge\bfseries}{}{0em}{\makebox[17mm][l]{\thechapter}\hangindent=\leftskip}
+\titleformat{\section}{\Large\bfseries}{}{0em}{\makebox[17mm][l]{\thesection}\hangindent=\leftskip}
+\titleformat{\subsection}{\bfseries}{}{0em}{\makebox[17mm][l]{\thesubsection}\hangindent=\leftskip}
 \titleformat{\subsubsection}{\itshape}{\thesubsubsection}{1em}{%
     \makebox[17mm][l]{\thesubsubsection}\hangindent=17mm
```

Comment:
It seems like you forgot to update the subsubsection rule.

Reply:
Yes, you right. I don't think this is used in Myxa or Komar documents. I can add this change.

Reply:
Fixed in commit [a733306](https://github.com/Zubax/document_templates/pull/6/commits/a733306e2a2c131fdfda8f469f770e3c3ee14846).